public/types.ts

```diff
@@ -34,12 +34,18 @@ export interface SecurityPluginStartDependencies {
 }
 
 export interface AuthInfo {
-  user_name: string;
-  tenants: {
+  user_name?: string;
```

Comment:
Why is the optional now?

Reply:
Done while testing something, removing it now. Thnaks.

Reply:
Was done while testing, fixing now.